conjur/wrapper/http_wrapper.py

```diff
@@ -55,11 +53,19 @@ def invoke_endpoint(http_verb, endpoint, params, *args, check_errors=True,
     request_method = getattr(requests, http_verb.name.lower())
 
     #pylint: disable=not-callable
-    response = request_method(url, *args,
-                              params=query,
-                              verify=ssl_verify,
-                              auth=auth,
-                              headers=headers)
+    try:
+        response = request_method(url, *args,
+                                  params=query,
+                                  verify=True,
+                                  auth=auth,
+                                  headers=headers)
+    except requests.exceptions.SSLError:
+        response = request_method(url, *args,
```

Comment:
is this a fallback to self-signed certificate ? If yes - please add a comment in code for it